src/studio/src/designer/frontend/packages/schema-editor/src/components/Editor.tsx

```diff
@@ -9,9 +9,9 @@ import type {
   ILanguage,
   ISchema,
   ISchemaState,
-  ObjectKind,
-  UiSchemaItem,
+  IUiSchemaItem,
```

Comment:
I don't see why we still add the `I` before the interfaces as it is C# semantics and not TypeScript semantics.
https://medium.com/@jonyeezs/hey-typescript-wheres-my-i-prefixed-interface-cd70efa10bd1

Reply:
Personal preference I guess, but I see your point. I have reverted the change to this interface - I suggest you bring this up f.ex. in an upcoming tech forum so we can decide on some common guidelines.